performance/src/main/java/performance/list/PerformanceTestVector.java

```diff
@@ -0,0 +1,26 @@
+package performance.list;
+
+import org.openjdk.jmh.annotations.*;
+import performance.TestData;
+
+import java.util.Arrays;
+import java.util.Vector;
+import java.util.concurrent.TimeUnit;
+
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+@BenchmarkMode(Mode.AverageTime)
+@Measurement(iterations = 5)
+@Warmup(iterations = 2)
+@State(Scope.Benchmark)
+public class PerformanceTestVector extends PerformanceTestList<Vector> {
+    @Param({"100", "1000", "1000000"})
+    int size;
+
+    @Setup(Level.Iteration)
+    public void setUp() {
+        index = size/2;
```

Comment:
index = size/2; why /2 ?

Reply:
because it is average element in collection